pymongo/encryption_options.py

```diff
@@ -140,6 +142,12 @@ def __init__(
             Or to supply a client certificate::
 
               kms_tls_options={'kmip': {'tlsCertificateKeyFile': 'client.pem'}}
+          - `csfle_path` (optional): Override the path to load the CSFLE library.
+          - `csfle_required` (optional): If 'true', refuse to continue encryption without a CSFLE
+          library
```

Comment:
I think this needs to align with the backtick on the line above, no the hyphen

Reply:
Fixed.